### ISSUE TITLE
Tableau de valeur et valeur exactes  /  Fraction and exact values error

### ISSUE THREAD

**Issue by glyj** [numworks/epsilon#2114]
#### Description du bug
Une fonction rationnelle du type x |-> (3x-4)/(x^2+1)  donnera f(-1/3)=-9/2 dans le module de calcul, mais dans le grapheur avec le mode calcul exact donnera une fraction du type -(499999 000000/111 11 08889)

The function f: x |-> (3x-4)/(x^2+1)  gives f(-1/3)=-9/2 in the calculus app, but with the graph app (exact results)  it gives  -(499999 000000/111 11 08889)

#### Captures d'écran
  
![image](https://github.com/numworks/epsilon/assets/54182510/a18514b4-1116-4aff-b7de-2f05411a5de0)

![image](https://github.com/numworks/epsilon/assets/54182510/3c1641c8-d5e5-4541-8ff2-bc108bcf6945)

#### Reproduction
Étapes pour reproduire l'erreur:
1. dans l'application grapheur : entrer la fonction x|-> (3x-4)/(x^2+1)
2. dans l'onglet Tableau, choisir la valeur -1/3 pour x
3. activer les résultats exacts
4. on obtient une fraction du type - 49999000000/1111110888889

#### Expected behavior
résultat attendu : -9/2

expected result : -9/2

#### Environment
 - Epsilon version (21.3.0).
 - Émulateur, calculatrice à jour N0120

**Comment by artaxxx** [comment]
Indeed, it's because the value of x is not exact in the table. It's in our plans for the future
